packages/jade-coffee/package.js

```diff
@@ -1,19 +1,21 @@
 Package.describe({
   summary: "Jade template language with inline coffeescript and Javascript support",
-  version: "0.1.4",
+  version: "0.1.6",
   name: "xiphy:jade-coffee",
   git: "https://github.com/xiphias/meteor-jade-coffee.git",
   documentation: "../../README.md"
 });
 
 Package.registerBuildPlugin({
-  name: "compileJade",
+  name: "compileJadeBatch",
```

Comment:
Please don't merge handler.js in this PR, as it's a bigger structural
change.

On Fri, Oct 30, 2015 at 4:43 PM, Damon Blais <notifications@github.com>
wrote:

> In packages/jade-coffee/package.js
> <https://github.com/xiphias/meteor-jade-coffee/pull/6#discussion_r43516927>
> :
>
> >    name: "xiphy:jade-coffee",
> >    git: "https://github.com/xiphias/meteor-jade-coffee.git",
> >    documentation: "../../README.md"
> >  });
> >
> >  Package.registerBuildPlugin({
> > -  name: "compileJade",
> > +  name: "compileJadeBatch",
>
> Can do; noticed an issue though: handler.js wasn't merged, I'm trying to
> merge it now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xiphias/meteor-jade-coffee/pull/6/files#r43516927>.
>